tests/Cms/Pages/PagesTest.php

```diff
@@ -299,11 +299,188 @@ public function testFindByIdAndUri()
         $this->assertIsPage($site->children()->findByUri('grandma/mother'), 'grandma/mother');
         $this->assertIsPage($site->children()->findByUri('grandma/mother/'), 'grandma/mother');
         $this->assertIsPage($site->children()->findById('grandma/mother.json'), 'grandma/mother');
+        $this->assertIsPage($site->children()->findById('grandma')->children()->findById('mother'), 'grandma/mother');
+        $this->assertIsPage($site->children()->findById('grandma')->children()->findById('grandma/mother'), 'grandma/mother');
+//         $this->assertIsPage($site->children()->findById('mother'), 'grandma/mother');
+//         $this->assertIsPage($site->children()->findByUri('mother'), 'grandma/mother');
         $this->assertIsPage($site->children()->findById('grandma/mother/child'), 'grandma/mother/child');
         $this->assertIsPage($site->children()->findById('grandma/mother/child/'), 'grandma/mother/child');
         $this->assertIsPage($site->children()->findByUri('grandma/mother/child'), 'grandma/mother/child');
         $this->assertIsPage($site->children()->findByUri('grandma/mother/child/'), 'grandma/mother/child');
         $this->assertIsPage($site->children()->findByUri('grandma/mother/child.json'), 'grandma/mother/child');
+        $this->assertIsPage($site->children()->findById('grandma/mother')->children()->findById('child'), 'grandma/mother/child');
+        $this->assertIsPage($site->children()->findById('grandma/mother')->children()->findById('grandma/mother/child'), 'grandma/mother/child');
+//         $this->assertIsPage($site->children()->findById('child'), 'grandma/mother/child');
+//         $this->assertIsPage($site->children()->findByUri('child'), 'grandma/mother/child');
+
+        $pages = new Pages($site->children()->find('grandma', 'grandma/mother', 'grandma/mother/child'));
+        $this->assertIsPage($pages->findById('grandma'), 'grandma');
+        $this->assertIsPage($pages->findById('grandma/mother'), 'grandma/mother');
+//         $this->assertIsPage($pages->findById('mother'), 'grandma/mother');
```

Comment:
This passes in a multilang environment.

Reply:
Could it be that this is, because in non-default language we end up with a `findBy('slug', $id)` call that just looks for a match of the slug itself. But with the default language, we are now stuck with `->get()` that needs the full ID, not just the slug?

Not sure here myself, my head doesn't wrap fully around this yet, but a hunch I have.

Reply:
Yes, that is accurate. What was confusing to me was why it was happening in the `en` language (which should be the default in the test case).

Turns out, in the test cases, `en` _wasn't_ actually being treated as a default language, which threw off the logic.

I've added `'default' => true,` which makes the multi-lang version constant to the single-language version.